python/kvikio/kvikio/_lib/remote_handle.pyx

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+
+# distutils: language = c++
+# cython: language_level=3
+
+from typing import Optional
+
+from cython.operator cimport dereference as deref
+from libc.stdint cimport uintptr_t
+from libcpp.memory cimport make_unique, unique_ptr
+from libcpp.string cimport string
+from libcpp.utility cimport move, pair
+
+from kvikio._lib.arr cimport parse_buffer_argument
+from kvikio._lib.future cimport IOFuture, _wrap_io_future, future
+
+
+cdef extern from "<kvikio/remote_handle.hpp>" nogil:
+    cdef cppclass cpp_RemoteEndpoint "kvikio::RemoteEndpoint":
+        pass
+
+    cdef cppclass cpp_HttpEndpoint "kvikio::HttpEndpoint":
+        cpp_HttpEndpoint(string url) except +
+
+    cdef cppclass cpp_RemoteHandle "kvikio::RemoteHandle":
```

Comment:
```suggestion
cdef extern from "<kvikio/remote_handle.hpp>" nogil namespace "kvikio":
    cdef cppclass RemoteEndpoint:
        pass

    cdef cppclass HttpEndpoint:
        HttpEndpoint(string url) except +

    cdef cppclass RemoteHandle:
```

?
Since `RemoteEndPoint`/`HttpEndpoint` and `RemoteHandle` are not used by any python cdef class names.

Reply:
I keep mixing C++ and Python objects when reading Cython, I think always using `cpp_` makes it a bit easier?

Reply:
Sure (nothing to do).